modules/ROOT/pages/clustering/setup/single-to-cluster.adoc

```diff
@@ -73,5 +78,79 @@ The last step is to enable the `Free` servers using `ENABLE SERVER` (see xref:cl
 
 Once all servers are enabled, you can scale up user databases using xref:clustering/databases.adoc#alter-topology[`ALTER DATABASE`], if desired.
 
-====
+======
+[role=include-with-Discovery-service-v2 label--new-5.23]
+======
+
+In this example, a standalone server named `server01` is running and two additional servers, `server02` and `server03`, are to be added to form a cluster.
+The two additional servers are configured according to xref:clustering/setup/deploy.adoc#cluster-example-configure-a-three-primary-cluster[Configure a cluster with three servers] and are _not_ running.
+Neo4j Enterprise Edition is installed on all three servers.
+
+Start by stopping the standalone server.
+Once it is stopped, edit the xref:configuration/file-locations.adoc[_neo4j.conf_] file to include the discovery endpoints of itself and the servers that will form the cluster.
+
+Starting with Neo4j 5.23, it is recommended to use v2 of the discovery service.
+For more details on discovery services, see xref:clustering/setup/discovery.adoc[].
+
+To use the discovery service v2:
+
+* You have to set the configuration of `dbms.cluster.discovery.version` to `V2_ONLY`.
+* Instead of `dbms.cluster.discovery.endpoints`, use `dbms.cluster.discovery.v2.endpoints`.
+
+The example below uses `dbms.cluster.discovery.resolver_type=LIST`.
+
+._neo4j.conf_ on server01.example.com:
+[source, properties]
+----
+server.default_listen_address=0.0.0.0
+server.default_advertised_address=server01.example.com
+dbms.cluster.discovery.v2.endpoints=server01.example.com:6000,server02.example.com:6000,server03.example.com:6000
+dbms.cluster.discovery.version=V2_ONLY
+initial.dbms.default_primaries_count=3
```

Comment:
Unrelated to this PR: There is no point setting any `initial.` setting when adding servers to a cluster that has already started, it's only picked up when the DBMS is created for the first time. This applies to the V1 tab too. I'd just remove this line.